app/views/chomp_sessions/_form.html.erb

```diff
@@ -28,49 +25,15 @@
 
   <label class="integer optional mb-3" for="chomp_session_invitees">Number of Friends (optional)</label>
   <div class="flex-between">
-    <a class="btn-success rounded-lg text-white incrementer minus d-flex align-items-center justify-content-center" data-offset="-1">-</a>
-    <span id="counter" data-count="1" class="font-weight-bold">1</span>
-    <a href="#" class="btn-success rounded-lg text-white incrementer d-flex align-items-center justify-content-center" data-offset="1" type="button">+</a>
+    <a class="round-bordered incrementer minus" data-chomp-target="deduct" data-action="click->chomp#decrement">-</a>
```

Comment:
can you add back "btn-success rounded-lg text-white"? don't like the bordered design cos we didn't use it else where (personal pref)